File: core/src/org/rexcellentgames/burningknight/debug/LightCommand.java
package org.rexcellentgames.burningknight.debug;

import org.rexcellentgames.burningknight.entity.level.LightLevel;

public class LightCommand extends ConsoleCommand {
	{
		name = "/light";
		shortName = "/lt";
	}

	@Override
	public void run(Console console, String[] args) {
		LightLevel.LIGHT = !LightLevel.LIGHT;
	}
}

File: core/src/org/rexcellentgames/burningknight/debug/LightCommand.kt
package org.rexcellentgames.burningknight.debug

import org.rexcellentgames.burningknight.entity.level.LightLevel

class LightCommand : ConsoleCommand("/light", "/lt", "Toggles light") {
  override fun run(console: Console, args: Array<String>) {
    LightLevel.LIGHT = !LightLevel.LIGHT
  }
}